### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by benlogan**
a few minor improvements to wording on the readme, including a typo or two...

### CODE REVIEW

**Comment by Henry-WattTime** [comment]
Thanks for making these improvements and catching these errors.

File: README.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ Electricity has a carbon intensity depending on where and when it is consumed. A
 - Check the [issues tab](https://github.com/Green-Software-Foundation/software_carbon_intensity/issues) for active and closed conversations regarding the spec.
 
 ## Contributing
-The recommended approach for getting invlved with the specification is to:
+The recommended approach for getting involved with the specification is to:
 - Read the [development version](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/dev/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md) of the specification.
-- Raise an issue, question, recommendation in the issues tab above and start a discussion with other members.
-- Once a form of agreement has been reached, then raise a pull request to update the specification with your recommended changes.
+- Raise an issue, question or recommendation in the issues tab above and start a discussion with other members.
```
Comment (atg-abhishek) [inline README.md:22]:
```suggestion
- Raise an issue, question, or recommendation in the issues tab above and start a discussion with other members.
```

**Review by atg-abhishek — CHANGES_REQUESTED** [review]
LGTM, added some more minor changes as suggestions.

**Comment by Henry-WattTime** [comment]
WG approved.

**Pull request merged**